### ISSUE TITLE
Payment service works wrong with version 0.3.x

### ISSUE THREAD

**Issue by giving-sesame** [joue-quroi/spoof-timezone#29]
Hi. I use payment service portmone.com.ua to pay for utility bills. There is a dashboard with the possibility to choose the period of bills. After your addon is updated to any version 0.3.x, the choice of period works in wrong way: for example, if I select November, I get October. With version 0.2.7 everything works correct.

**Comment by joue-quroi** [comment]
I am not sure how to debug the issue due to the language difference and login requirements. Do you have this problem on another website too? Or can you provide steps to reproduce?

**Comment by giving-sesame** [comment]
Yes, I understand the difficulties. This is the only website where I faced such a problem.

I'll try to do some research by myself. By now I noticed the only difference in network requests. When version 0.3 is installed, the browser requests the current period in the following way:

`https ://www.portmone.com.ua/api/v1/b2c/bills/bill?startDate=30.09.2022&endDate=31.10.2022&status=CREATED,PAYED,REJECTED,RETURN,CANCELED`

When version 0.2 is installed:

`https://www.portmone.com.ua/api/v1/b2c/bills/bill?startDate=01.10.2022&endDate=01.11.2022&status=CREATED,PAYED,REJECTED,RETURN,CANCELED`

I'll do more experiments

**Comment by joue-quroi** [comment]
If you are on Chrome, go to the network tab and find the initiator for this request. Then set a breakpoint on the method building this URL. Either there is a method that the extension is not overwriting correctly, or somehow the date object is the native one.

**Comment by giving-sesame** [comment]
I found a solution by accident. While experimenting I changed timezone in options, and the website started to work normally. I changed back to my usual timezone and it still works.

I have no idea what happened. Maybe you can add this way to your troubleshooting guide :)
